components/wmtk_components/json_utils.hpp

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include <filesystem>
+#include <nlohmann/json.hpp>
+
+namespace nlohmann {
+template <>
+struct adl_serializer<std::filesystem::path>
```

Comment:
What do we need that for? Please add some documentation explaining what it is and where it is used.